crates/torii/core/src/processors/store_update_member.rs

```diff
@@ -29,16 +26,7 @@ where
         "StoreUpdateMember".to_string()
     }
 
-    fn validate(&self, event: &Event) -> bool {
```

Comment:
Why did we remove the validate?

Reply:
Not used anymore as in all the other processors since the deserialization is auto generated now.